akka-http-testkit/src/main/resources/reference.conf

```diff
@@ -0,0 +1 @@
+akka.http.testkit.marshalling.timeout = 1 s
```

Comment:
We usually prefix those with a little bit of docs comment giving an explanation, in this case maybe it is evident what it does from the name, so not sure it is needed. Add if you agree or leave it be if not :)

Reply:
Ouch I totally missed your comment! I think it's pretty clear though, I cannot think of any documentation that wouldn't just repeat the name of the param